devpi_ldap/main.py

```diff
@@ -175,7 +175,7 @@ def _search(self, conn, config, **kw):
             if any(attribute_name in x.get('attributes', {}) for x in conn.response):
                 def extract_search(s):
                     if 'attributes' in s:
-                        return s['attributes'][attribute_name]
+                        return [s['attributes'][attribute_name]]
```

Comment:
You're right, it must be an ldap3 bug. I receive a list in `raw_attributes` but not in `attributes`, eg:
```
{'attributes': {u'distinguishedName': u'CN=James Laird-Wah,OU=XXX,DC=XXX'},
 'dn': u'CN=James Laird-Wah,OU=XXX,DC=XXX',
 'raw_attributes': {u'distinguishedName': ['CN=James Laird-Wah,OU=XXX,DC=XXX']},
 'type': 'searchResEntry'}
```

This conflicts with the documentation which describes 'attributes: a dictionary of returned attributes and their values. Values are list. Values are in UTF-8 format'